### PR TITLE
add default id for MKS TS35 V2.0

### PR DESCRIPTION
**Pull request by ellensp**
### Description

The latest batch of MKS TS35 V2.0 TFT's do not respond with their ID when they are asked for them. 
TFT_COLOR_UI need this ID, without it you get a full white screen.
If the both attempts to read the LCD ID fails and TFT_DEFAULT_DRIVER is defined, it will now use that value.

### Requirements
```cpp
#define SERIAL_PORT -1
#define MOTHERBOARD BOARD_MKS_ROBIN_NANO_V3
#define MKS_TS35_V2_0
#define TFT_COLOR_UI
```
### Benefits

TFT display works as expected.

### Related Issues
Issue https://github.com/MarlinFirmware/Marlin/issues/22003

### CODE REVIEW

File: Marlin/src/inc/Conditionals_LCD.h [hunk 1]
```diff
@@ -1141,6 +1141,7 @@
  *  - GRAPHICAL_TFT_UPSCALE
  */
 #if ENABLED(MKS_TS35_V2_0)          // Most common: ST7796
+  #define TFT_DEFAULT_DRIVER 0x7796 // ST7796
```
Comment (rhapsodyv) [inline Marlin/src/inc/Conditionals_LCD.h]: You can use the driver define ST7796

Comment (ellensp) [reply Marlin/src/inc/Conditionals_LCD.h]: no, it is not available error: 'ST7796' was not declared in this scope
and I didn't want to include Marlin/src/lcd/tft_io/tft_io.h for 1 definition

Comment (delaosa) [reply Marlin/src/inc/Conditionals_LCD.h]: I already tested  #define TFT_DRIVER ST7796 and it works.

**Comment by rhapsodyv** [comment]
This a very good addition. I think we should add this changes.

**Pull request merged**